src/PublicKey/Rsa.php

```diff
@@ -314,7 +318,11 @@ public function decrypt(
                 break;
         }
 
-        return $key->decrypt($data);
+        if (null === $this->getOptions()->getPadding()) {
```

Comment:
Don't repeat the call chain to `$this->getOptions()->getPadding()`, as it is not guaranteed to return the same values over subsequent calls anyway. Assign it to a variable instead.